lib/tmuxinator/assets/template.erb

```diff
@@ -19,18 +19,7 @@ cd <%= root || "." %>
   # Run on_project_first_start command.
   <%= hook_on_project_first_start %>
 
-  # Create the session and the first window. Manually switch to root
-  # directory if required to support tmux < 1.9
-  <% if windows.first.root? %>
-    <% if Tmuxinator::Config.version < 1.9 %>
   TMUX= <%= tmux_new_session_command %>
-  <%= windows.first.tmux_window_command_prefix %> <%= "cd #{windows.first.root}".shellescape %> C-m
-    <%- else -%>
-  TMUX= <%= tmux_new_session_command %> -c <%= windows.first.root.shellescape %>
-    <% end %>
-  <%- else -%>
-  TMUX= <%= tmux_new_session_command %>
-  <% end %>
```

Comment:
All this was trying to set up the session path without breaking old tmux versions. The logic is mostly the same, but it's moved into `project.rb`.